cmd/internal/pipelinetests/testdata/target_scrape_and_write_otel_conversion.river

```diff
@@ -0,0 +1,47 @@
+logging {
+  level  = "debug"
+  format = "logfmt"
+}
+
+// Scrape test target and send to OTEL converter
```

Comment:
Example of a test that touches on quite a lot of components from `prometheus` and `otelcol` namespaces, testing the agent as a whole. We can quite easily go further with relabels etc. 

The goal is not to make an exhaustive test, but to verify that the pipeline works and data flows through it really.